src/controllers/__tests__/sessionLockController.test.ts

```diff
@@ -45,7 +103,7 @@ describe("SessionLockController#lockUserSession", () => {
       mockRedisUserMetadataStorage
     );
 
-    const response = await controller.lockUserSession(req);
+    const response = await controller.getUserSession(req);
```

Comment:
Why this change?

Reply:
reverted in [765d30d](https://github.com/pagopa/io-backend/pull/929/commits/765d30d4bac578e073586619f853242ce7b5d0c4)